packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/index.js

```diff
@@ -130,86 +135,69 @@ const ApplicationInfosPage = () => {
                   })}
                 </Typography>
 
-                <Grid paddingTop={1}>
+                <Grid gap={5} as="dl">
                   <GridItem col={6} s={12}>
-                    <Typography variant="sigma" textColor="neutral600">
+                    <Typography variant="sigma" textColor="neutral600" as="dt">
                       {formatMessage({
                         id: 'Settings.application.strapiVersion',
                         defaultMessage: 'strapi version',
                       })}
                     </Typography>
-                    <Typography as="p">v{strapiVersion}</Typography>
-                    <Link
-                      href={
-                        appInfos.communityEdition
-                          ? 'https://discord.strapi.io'
-                          : 'https://support.strapi.io/support/home'
-                      }
-                      isExternal
-                      endIcon={<ExternalLink />}
-                    >
-                      {formatMessage({
-                        id: 'Settings.application.get-help',
-                        defaultMessage: 'Get help',
-                      })}
-                    </Link>
+                    <Typography as="dd">v{strapiVersion}</Typography>
+                    <Box paddingTop={3}>
+                      {shouldUpdateStrapi && (
+                        <Link
```

Comment:
Would it be possible to move the <Link> into the `dd` Typography element? This would ensure 1) the markup is valid 2) it is part of the defined term for screenreaders.

Reply:
You could say:

```
Flex gap=3 as="dd"
  <Typography as="dd">v{strapiVersion}</Typography>
  <Box>
```

to make it less weird?